druid/src/test/java/org/apache/calcite/adapter/druid/DruidQueryFilterTest.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.adapter.druid;
+
+import org.apache.calcite.rel.type.RelDataType;
+import org.apache.calcite.rel.type.RelDataTypeField;
+import org.apache.calcite.rex.RexCall;
+import org.apache.calcite.rex.RexInputRef;
+import org.apache.calcite.rex.RexLiteral;
+import org.apache.calcite.rex.RexNode;
+import org.apache.calcite.rex.RexVisitor;
+import org.apache.calcite.sql.SqlKind;
+import org.apache.calcite.sql.type.SqlTypeFamily;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.google.common.collect.Lists;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.io.IOException;
+import java.io.StringWriter;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+/**
+ * Test class for druid filter generation
+ */
+public class DruidQueryFilterTest {
+
+  @Test
+  public void testInFilter()
+      throws NoSuchMethodException, InvocationTargetException, IllegalAccessException,
+          IOException {
+    RelDataType relDataType = Mockito.mock(RelDataType.class);
+    RelDataTypeField relDataTypeField = Mockito.mock(RelDataTypeField.class);
+
+    DruidQuery.Translator translator = new DruidQuery.Translator(null, relDataType);
+    Method translateFilter = DruidQuery.Translator.class.getDeclaredMethod("translateFilter",
+            RexNode.class
+    );
+    RexNode[] rexNodes = new RexNode[1];
+    RexCall rexNode = Mockito.mock(RexCall.class);
```

Comment:
I think that it is easier (and more reliable) to create the RexNode objects instead of mocking them (as in ```org.apache.calcite.rex.RexExecutorTest```). For instance, if I am not mistaken, by calling ```rexBuilder.makeCall(SqlStdOperatorTable.BETWEEN, ...```, the validator would have caught the issue with the BETWEEN operator.